types/sparkly/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for sparkly 3.1
+// Project: https://github.com/sindresorhus/sparkly
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = sparkly;
+
+declare function sparkly(numbers: any[], options?: sparkly.Options): string;
```

Comment:
This is what I thought first, too. But then I discovered this in the example section:
```
// specifying anything other than finite numbers will cause holes
sparkly([0, 3, 5, '', 4, 3, 4, 10]);
//=> '▁▃▄ ▄▃▄█'
```
So this is by design to specify holes. I was just not sure whether I should allow any type as suggested by the comment or only numbers and strings. Internally the code does an `isFinite()` check.